server/state.go

```diff
@@ -318,6 +313,7 @@ func (s *SiteVisitController) HandleStatusPhase() {
 			// send the status update to the user immediately.
 			if response != nil {
 				u.Message(response)
+				u.Message(NewSetClockMessage(SiteVisitStatusDuration))
```

Comment:
Could you verify if this is the right place in the nested if statements to send set_clock?